libs/core/src/lib/popover/popover-mobile/popover-mobile.component.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { A11yModule } from '@angular/cdk/a11y';
+import { OverlayModule } from '@angular/cdk/overlay';
+import { Component, Inject, InjectionToken, ViewChild } from '@angular/core';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { MobileModeConfig } from '@fundamental-ngx/core';
+import { DialogRef } from '@fundamental-ngx/core/dialog';
+import { PopoverComponent, PopoverModule } from '../public_api';
+import { PopoverMobileComponent } from './popover-mobile.component';
+import { PopoverMobileModule } from './popover-mobile.module';
+
+const MOBILE_CONFIG_TEST_TOKEN = new InjectionToken<MobileModeConfig>('For test purposes');
+const MOBILE_CONFIG: MobileModeConfig = { title: 'Test popover title' };
+
+@Component({
+    template: `
+        <fd-popover #popoverMobile
+            [mobile]="true"
+            [mobileConfig]="{ title: 'Mobile Popover Test', hasCloseButton: true }"
+            style="display: block">
```

Comment:
why do you have this?

Reply:
`menu-mobile` and `select-mobile` also use this approach, these components use service injected on component level instead of module level, to test service method calls this is the only approach I could make work